config/config.go

```diff
@@ -702,6 +705,7 @@ func NewConfig() (c *Config) {
 			CacheIstioTypes:             []string{"AuthorizationPolicy", "DestinationRule", "EnvoyFilter", "Gateway", "PeerAuthentication", "RequestAuthentication", "ServiceEntry", "Sidecar", "VirtualService", "WorkloadEntry", "WorkloadGroup"},
 			CacheNamespaces:             []string{".*"},
 			CacheTokenNamespaceDuration: 10,
+			TokenExpireDuration:         90 * 86400, // Default to 90 days
```

Comment:
A little note:

So, the token will be re-read after 90 days. In the {k8s changelog for v1.21](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.21.md) I see this:

```
The BoundServiceAccountTokenVolume feature has been promoted to beta, and enabled by default.
    Clients should reload the token from disk periodically (once per minute is recommended) to ensure they continue to use a valid token. k8s.io/client-go version v11.0.0+ and v0.15.0+ reload tokens automatically.
```

So, the recommendation is to re-read every minute.
I'm not wrong with keeping 90 days. I just want to be sure that you really want 90 days as the default value, because my thought is that 90days may be a default for EKS, while other flavors of k8s may have something different (although I may be wrong).

I think you also may need an additional PR on the [kiali/kiali-operator repo](https://github.com/kiali/kiali-operator/) to, and probably also the [kiali/helm-charts repo](https://github.com/kiali/helm-charts/) to document the new setting and let people change this value.

Reply:
Thanks for the review, @israel-hdez ! I will have a look

Reply:
New configs sometimes trigger a change in the `kiali-server` chart: https://github.com/kiali/helm-charts/tree/master/kiali-server

But most often, no changes are required. So, you may simply try the `kiali-server` chart and try to add your new config. If it gets configured correctly, then no changes are needed there. There is also the [`kiali-operator` chart](https://github.com/kiali/helm-charts/tree/master/kiali-operator) but that one very rarely needs a change.